### ISSUE TITLE
what is this?

### ISSUE THREAD

**Issue by charlieporth1** [diorcety/mazda3-pyds#1]
what is this?

**Comment by mrg0lden** [comment]
@diorecty Will there be any kind of docs?

**Comment by diorcety** [comment]
No. If you don't understand what is done here, you should not use it. What do you want use it?

**Comment by mrg0lden** [comment]
@diorcety We have a cx-9 that doesn't have the lock button; thus, it doesn't have the ability to change auto-lock mode, which is disabled by the way. I thought these files could contain something that helps with this problem.

**Comment by diorcety** [comment]
Yes the code can  do that, but use the forscan method  it will be easier if you don't understand this code and how it works
